test/lib/util.js

```diff
@@ -880,3 +880,48 @@ describe('Utils buildSourceUrl', () => {
     expect(result).to.eq('https://pypi.org/project/zuul/3.3.0/')
   })
 })
+
+describe('normalizeLicenseExpression', () => {
+  it('should normalize license', () => {
+    const expression = 'MIT AND GPL-3.0'
+    const result = utils.normalizeLicenseExpression(expression)
+    expect(result).to.eq('MIT AND GPL-3.0')
+  })
```

Comment:
This particular translation keeps getting incorrectly changed to a LicenseRef in the scancode map when the map update workflow runs (see PR #1207).  I want to be sure there is a test that will fail for the PR that updates the map if this is translated to a LicenseRef the next time the workflow runs.  
```suggestion
  })
  it('should normalize license to SPDX equivalent', () => {
    /* 
      NOTE: If this fails in tests for generated scancode map workflow PR, it is incorrect if it is expecting a LicenseRef.  
      There is an SPDX valid license which does not require a LicenseRef meaning this test is correct as is.
    */
    const expression = 'net-snmp'
    const result = utils.normalizeLicenseExpression(expression)
    expect(result).to.eq('Net-SNMP')
  })
```